test/unit_testing/service/movie_api_service_unit_test.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:http/http.dart' as http;
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:movie_db_app/src/core/util/constants.dart';
+import 'package:movie_db_app/src/data/datasource/remote/movie_list_api_service.dart';
+
+import 'movie_api_service_unit_test.mocks.dart';
+
+@GenerateNiceMocks([MockSpec<MovieListApiService>(), MockSpec<http.Client>()])
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
```

Comment:
You don't need this method call in this case. Same above

Reply:
solved